hera_cal/tests/test_utils.py

```diff
@@ -0,0 +1,18 @@
+import nose.tools as nt
+from hera_cal.utils import get_HERA_aa
+import numpy as np,sys
+from hera_cal.calibrations import CAL_PATH
+freqs = np.array([0.15])
+
+class Test_utils(object):
```

Comment:
Added a test to check the two "epochs" we currently have, H19 and ~H34.

Note: antpos_ideal is currently constructed with "filler" antennas at position -1,-1,-1.  omnical, firstcal all automatically ignore these fillers but other things probably wont.